nextmv/cloud/application.py

```diff
@@ -631,10 +646,7 @@ def new_run_with_result(
             polling_options=polling_options,
         )
 
-    def run_input(
-        self,
-        run_id: str,
-    ) -> Dict[str, Any]:
+    def run_input(self, run_id: str) -> Dict[str, Any]:
```

Comment:
Our formatters seem to be configured differently. Or, was that a deliberate choice? :thinking:

Reply:
They should be configured the same, it was just a deliberate choice of the line being so short that each param doesn’t need to go on  a new line.

Reply:
Makes sense. :blush: